waldur-site-agent-config.yaml

```diff
@@ -0,0 +1,37 @@
+sentry_dsn: "" # Data Source Name for Sentry (more info https://docs.sentry.io/product/sentry-basics/dsn-explainer/).
```

Comment:
is this default config file? I think then better to take it from upstream / develop branch